cmd/ulxly/ulxly.go

```diff
@@ -126,6 +182,294 @@ var DepositsCmd = &cobra.Command{
 	},
 }
 
+//go:embed depositNewUsage.md
+var depositNewUsage string
+var depositNewCmd = &cobra.Command{
+	Use:     "deposit-new",
+	Short:   "Make a uLxLy deposit transaction",
+	Long:    depositNewUsage,
+	Args:    cobra.NoArgs,
+	PreRunE: checkDepositArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+		// Dial the Ethereum RPC server.
+		client, err := ethclient.DialContext(ctx, *ulxlyInputArgs.DepositRPCURL)
+		if err != nil {
+			log.Error().Err(err).Msg("Unable to dial rpc")
+		}
+		defer client.Close()
+
+		bridgeV2, err := ulxly.NewUlxly(common.HexToAddress(*ulxlyInputArgs.DepositBridgeAddress), client)
+		if err != nil {
+			return err
+		}
+
+		privateKey, err := crypto.HexToECDSA(*ulxlyInputArgs.PrivateKey)
+		if err != nil {
+			log.Error().Err(err).Msg("Unable to retrieve private key")
+		}
+
+		publicKey := privateKey.Public()
+		publicKeyECDSA, ok := publicKey.(*ecdsa.PublicKey)
+		if !ok {
+			log.Error().Msg("Error casting public key to ECDSA")
+		}
+
+		fromAddress := crypto.PubkeyToAddress(*publicKeyECDSA)
+
+		value := big.NewInt(*ulxlyInputArgs.Value)
+		gasLimit := *ulxlyInputArgs.GasLimit // Deposit transactions seem to use >300,000 units of gas to be safe.
```

Comment:
For something like this, I would probably be okay with estimating gas, but this works too!

Reply:
Initially, manual gaslimit setting seemed quicker to implement, but I agree gas estimation should be there as well